graphql/schema/types/project_settings.graphql

```diff
@@ -15,6 +15,7 @@ enum ProjectSettingsSection {
   PERIODIC_BUILDS
   PLUGINS
   CONTAINERS
+  UI
```

Comment:
I noticed that this page is labeled "Default Filters" in the sidebar in the designs. Are you thinking of renaming it to "UI Settings" or something more generic? If not it'd probably be helpful for the enum to align with the page name.

Reply:
Oops I missed Mohamed's comment above 😅 I think `VIEWS_AND_FILTERS` is a reasonable name tbh. Mostly I think alignment between the page names here and on Spruce is important.